### ISSUE TITLE
forbidCustomFunctions does not recognize strings used as callbacks

### ISSUE THREAD

**Issue by mabar** [shipmonk-rnd/phpstan-rules#287]
Forbidden functions are not recognized in cases like this:

```php
array_map('gettext', $array);
```

Configuration:

```neon
parameters:
	shipmonkRules:
		enableAllRules: false

		forbidCustomFunctions:
			enabled: true
			list:
				'gettext': 'Use t() instead.'
				'_': 'Use t() instead.'
```

I haven't checked other callable syntaxes like `[$object, 'method']` but they may be problem as well

**Comment by janedbal** [comment]
Thanks, fixed (along with other improvements) in [4.1.0](https://github.com/shipmonk-rnd/phpstan-rules/releases/tag/4.1.0)

**Comment by mabar** [comment]
Well, that was fast. Thank you!
